Detectors/ITSMFT/ITS/macros/test/CheckTracks.C

```diff
@@ -305,8 +305,12 @@ void CheckTracks(std::string tracfile = "o2trac_its.root", std::string clusfile
       }
 
       Int_t mID = mcTrack.getMotherTrackId();
-      if (mID >= 0)
-        continue; // Select primary particles
+      if (mID >= 0) {
+        const auto& mom = (*mcArr)[mID];
+        int pdg = std::abs(mom.GetPdgCode());
+        if (pdg > 100 || (pdg < 20 && pdg > 10))
+          continue; // Select primary particles
+      }
       Int_t pdg = mcTrack.GetPdgCode();
       if (TMath::Abs(pdg) != 211)
```

Comment:
in this case, should not this be eliminated

Reply:
The check on pions? We can remove it